### PR TITLE
Solnyshko Update

### PR DESCRIPTION
**Pull request by master832**
## About The Pull Request

Hi everyone,
Trading station Solnyshko received desing disk containing super and hyper series cells in its hidden inventory. Twice as rare as advanced tool disk.
Super series cells have moved to the Asters Cells II tab, similar to powermat (Asters Guild Power-Mat), in which the cells are contraband. All prices remain unchanged.
I also changed position of sink in the Chemistry Lab so that it is not at floor level.
robustcell
Update: Disk Designs II tab and Aster's Advanced Robustcells disk have been removed.

## Why It's Good For The Game

Disc allows you to save money at expense of resources when printing cells. Like any design disk.
Hi-tier series cells from new disk will be of interest to technomancers with thir nanoforge. This will allow not so much to depend on scientists in this regard. It's good when a thing can be obtained in several ways (nanoforge upgraded hi-tech cells).
Since hi-tier cells are already sold at this station, this will not be a hard imbalance.
Disk Designs II tab looks strange with one product, but maybe in future there will be something to add here.

## Testing

Game works as usual.
com

## Changelog
:cl:
tweak: Super series cells have moved to Asters Cells II tab on station Solnyshko
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/design_disks/asters.dm [hunk 1]
```diff
@@ -123,3 +123,18 @@
...
+	rarity_value = 22
+	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED_COMMON
+	license = 10
+	designs = list(
+		/datum/design/autolathe/cell/large/super,
+		/datum/design/autolathe/cell/large/hyper = 2,
+		/datum/design/autolathe/cell/medium/super,
+		/datum/design/autolathe/cell/medium/hyper = 2,
+		/datum/design/autolathe/cell/small/super,
+		/datum/design/autolathe/cell/small/hyper = 2
```
Comment (SirRichardFrancis) [inline code/game/objects/items/weapons/design_disks/asters.dm:139]:
```suggestion
		/datum/design/autolathe/cell/large/hyper = 2,
		/datum/design/autolathe/cell/medium/hyper = 2,
		/datum/design/autolathe/cell/small/hyper = 2
```

**Review by SirRichardFrancis — COMMENTED** [review]
I think that's a bad idea. Cool gif though.

**Comment by Humonitarian** [comment]
I agree with kirov here, Guild was meant to burn money with its orders, and this will also expand the Xcell supply, which on paper harms Moebius as well.

Cool gif though.

**Comment by Alliostra** [comment]
Harming Moebius is always morally correct. Cool gif.

**Comment by master832** [comment]
This is a controversial idea, so I removed some of code that harms Moebius. Because of this, PR changes almost nothing in game.

**Review by Firefox13 — APPROVED** [review]
Make them **PAY** (literally)

**Comment by TheShown911** [comment]
I think it's not a good idea and fox is cracked out of his mind.

**Review by hyperioo — COMMENTED** [review]
I don't know how the code was initially but now from what I understand it makes those super cells harder to get since they're hidden in Asters II instead of being readily available, so seems fine?

**Pull request merged**